src/frontend/components/text-button.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+  export let text : string;
+  export let clickFunction : () => void;
+  export let disabled : boolean = false;
```

Comment:
Not really relevant but eslint will likely complain about "irrelevant" typing (`var = false` implies it is boolean)

Reply:
Surprisingly enough, it isn't complaining for me. I don't see it as a problem though - being explicit about types has never hurt anyone. Perhaps we can reevaluate this lint rule later.